src/libs/actions/PersonalDetails.js

```diff
@@ -109,6 +110,7 @@ function formatPersonalDetails(personalDetailsList) {
             timezone,
             payPalMeAddress,
             phoneNumber,
+            avatarHighResolution,
```

Comment:
I think it would be better to use the `avatarThumbnail` and `avatar` properties. which means use `avatarThumbnail` instead of avatar and store `details.avatar` in the `avatar`. 

Now use `avatarThumbnail` where the avatar was used.

Reply:
> Now use avatarThumbnail where the avatar was used.

Do we want to do this change here? This might require massive code refactoring.

Reply:
I was expecting it to be done here because of the keys. But I don't mind another PR doing that. At last, I am fine with this as well if @Julesssss is.